src/src/CustomBuild/ESPEasyDefaults.h

```diff
@@ -250,6 +250,15 @@
 #ifndef DEFAULT_MQTT_RETAIN
 #define DEFAULT_MQTT_RETAIN                     false   // (true|false) Retain MQTT messages?
 #endif
+
+#ifndef DEFAULT_MQTT_DELETE_OLDEST
+#define DEFAULT_MQTT_DELETE_OLDEST              false  // (true|false) to delete oldest message when queue is full
```

Comment:
This "delete oldest" is not specific to MQTT.
It is just a queue management flag for any controller.

Reply:
renamed to DEFAULT_CONTROLLER_DELETE_OLDEST